charts/kafka-operator/crds/operator-kafka-crd.yaml

```diff
@@ -499,6 +499,47 @@ spec:
                           type: string
                       type: object
                     type: array
+                  envoy:
```

Comment:
is there a need to set required field for envoy?

Reply:
No, The field is not mandatory. As the description mention, it is applicable only if `EnvoyConfig.dedicatedEnvoyPerBrokerGroup` is set to true.